docs/tutorial/part-four/index.md

```diff
@@ -4,103 +4,84 @@ typora-copy-images-to: ./
 
-Data can also live in file types like Markdown, CSV, etc. as well as databases
-and APIs of all sorts.
+डेटा, फाइल्स जैसे Markdown, CSV आदि के साथ-साथ डेटाबेस और सभी प्रकार के APIs में भी रह सकती है।
 
-**Gatsby's data layer lets you pull data from these (and any other source)
-directly into your components**—in the shape and form you want.
+**Gatsby की डेटा लेयर आपको इन (और किसी भी अन्य सोर्स) से डेटा पुल करने देती है सीधे अपने कौम्पोनॅन्ट में** - जिस आकार और रूप में आप चाहते हैं।
 
-## Using Unstructured Data vs GraphQL
+## असंरचित डेटा बनाम GraphQL का उपयोग करना
 
-### Do I have to use GraphQL and source plugins to pull data into Gatsby sites?
+### क्या मुझे Gatsby साइटों में डेटा पुल करने के लिए GraphQL और सोर्स प्लगइन्स का उपयोग करना होगा?
 
-Absolutely not! You can use the node `createPages` API to pull unstructured data into Gatsby pages directly, rather than through the GraphQL data layer. This is a great choice for small sites, while GraphQL and source plugins can help save time with more complex sites.
+बिलकुल नहीं! आप GraphQL डेटा लेयर के बजाय सीधे Gatsby पेजेज में असंरचित डेटा को पुल करने के लिए नोड `createPages` API का उपयोग कर सकते हैं। यह छोटी साइटों के लिए एक बढ़िया विकल्प है, जबकि GraphQL और सोर्स प्लगइन्स अधिक जटिल साइटों के लिए समय बचाने में मदद कर सकते हैं।
 
-See the [Using Gatsby without GraphQL](/docs/using-gatsby-without-graphql/) guide to learn how to pull data into your Gatsby site using the node `createPages` API and to see an example site!
+अपने Gatsby साइट में नोड `createPages` API का उपयोग करके डेटा को कैसे पुल करे और एक उदाहरण साइट देखने के लिए, [बिना GraphQL के Gatsby का उपयोग करना](/docs/using-gatsby-without-graphql/) देखें!
 
-### When do I use unstructured data vs GraphQL?
+### मैं GraphQL बनाम असंरचित डेटा का उपयोग कब करूं?
 
-If you're building a small site, one efficient way to build it is to pull in unstructured data as outlined in this guide, using `createPages` API, and then if the site becomes more complex later on, you move on to building more complex sites, or you'd like to transform your data, follow these steps:
+यदि आप एक छोटी साइट का निर्माण कर रहे हैं, तो इसे बनाने के लिए एक कुशल तरीका है कि इस गाइड में उल्लिखित असंरचित डेटा को पुल करते हुए `createPages` API का उपयोग करके, और यदि बाद में साइट अधिक जटिल हो जाती है, आप अधिक जटिल साइटें का निर्माण करना चाहते है, या आप अपना डेटा बदलना चाहते हैं तो, इन स्टेप्स का अनुसरण करें:
 
-1.  Check out the [Plugin Library](/plugins/) to see if the source plugins and/or transformer plugins you'd like to use already exist
-2.  If they don't exist, read the [Plugin Authoring](/docs/creating-plugins/) guide and consider building your own!
+1. [प्लगइन लाइब्रेरी](/plugins/) देखें ताकि आप यह जान पाए की जिस स्रोत प्लगइन्स और/या ट्रांसफॉर्मर प्लगइन्स का आप उपयोग करना चाहते हैं वो पहले से ही मौजूद हैं या नहीं|
```

Comment:
`नहीं|` => `नहीं।`

Reply:
I do not see any difference here.

Reply:
<img width="109" alt="Screen Shot 2019-12-21 at 6 58 44 PM" src="https://user-images.githubusercontent.com/4654382/71308541-f3258880-2423-11ea-8289-fac7b53d0cbb.png">

Check the difference in purn-viram

Reply:
oh! yes..

Reply:
fixed